tests/test_subclass.py

```diff
@@ -183,8 +190,51 @@ def test_batch_subclass(database):
     }
     query = get_query(qgraph)
     output = list(database.run(query))[0]
-    assert len(output['results']) == 12
+    assert len(output['results']) == 15
     for result in output["results"]:
         for binding in result["node_bindings"]["n0"]:
             assert "qnode_id" in binding
             assert (binding["qnode_id"] == "HP:0000118") if binding["id"].startswith("HP") else (binding["qnode_id"] == "MONDO:0000001")
+
+def test_hierarchy_inference_on_superclass_queries(database):
```

Comment:
This test is a little goofy because in the real data, MONDO:0014488 would also be returned.  It's not worth changing exactly, but maybe make a comment explaining why?

Reply:
added a comment